src/home.py

```
@@ -0,0 +1,296 @@
+if "filter_option" not in st.session_state:
+    st.session_state.filter_option = "None"
+if "search_query" not in st.session_state:
+    st.session_state.search_query = ""
+
+st.set_page_config(page_title="Remail", layout="wide")
+
+#New Mail
+def add_email_form():
+    st.subheader("Compose New Email")
+    recipient = st.text_input("To:", key="recipient_input") 
+    cc = st.text_input("Cc:", key="cc_input")
```

Ruff (0.8.2)

Local variable `cc` is assigned to but never used

Remove assignment to unused variable `cc`

(F841)